packages/yoga/src/Card/native/PlanCard/List.jsx

```diff
@@ -51,7 +51,7 @@ const ListItem = withTheme(
               <Icon
                 width={16}
                 height={16}
-                fill={yogaTheme.yoga.colors.elements.selectionAndIcons}
+                fill={yogaTheme.yoga.colors.secondary}
```

Comment:
I think it would make more sense if we fill the icon with the same color as its next text (`yogaTheme.yoga.colors.text.primary`). What do you think?